Index: trunk/src/kbdrate.c
===================================================================
--- trunk.orig/src/kbdrate.c
+++ trunk/src/kbdrate.c
@@ -77,7 +77,6 @@
 
 #ifdef __sparc__
 #include <asm/param.h>
-#include <asm/kbio.h>
 #endif
 
 #ifndef KDKBDREP
Index: trunk/src/setleds.c
===================================================================
--- trunk.orig/src/setleds.c
+++ trunk/src/setleds.c
@@ -14,10 +14,6 @@
 #include "nls.h"
 #include "version.h"
 
-#ifdef __sparc__
-#include <asm/kbio.h>
-#endif
-
 static void
 usage(void)
 {
